servers/partner-ci/builders/vcenter.test.sh

```diff
@@ -27,14 +27,13 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
   fi
 fi
 
-if [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]]; then
-  export SNAPSHOTS_ID=$(cat snapshots.params | grep -Po '#\K[^ ]+')
-fi
-
-[ -z "${SNAPSHOTS_ID}" ] && { echo SNAPSHOTS_ID is empty; exit 1; }
+[[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
+  id=$(grep -Po '#\K[^ ]+' < snapshots.params)
+  export SNAPSHOTS_ID=$id
```

Comment:
export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params )
seems like this?